assets/component-article-card.css

```diff
@@ -138,3 +134,43 @@
     margin-right: 4rem;
   }
 }
+
+.article-card__image--small {
+  padding-bottom: 11rem;
+}
+
+.article-card__image--medium {
+  padding-bottom: 22rem;
+}
+
+.article-card__image--large {
+  padding-bottom: 33rem;
+}
```

Comment:
I could picture an artistic use of the small size, though you're right it likely won't be the most commonly used option. I don't have strong feelings. I'd love to reduce the code a bit, but I can't speak to how important that size option was to design @Oliviammarcello

Reply:
If we're keeping all those size options I'm ok with leaving it as is. Since you can choose to use medium or large if more `height` is needed 👍 
It just seemed like a niche use case.